### PR TITLE
Updates readme

### PR DESCRIPTION
**Pull request by irbekrm**
This PR:

- Adds functionality to generate CLI reference docs similar to https://github.com/cert-manager/cert-manager/tree/master/tools/cobra
- Adds generated reference docs
- Updates Readme, to add information on how to use jsctl to generate a 'Venafi TPP profile' Jetstack Secure components installation, removes some bits that I though users could discover from reference docs instead and moves the JSS user creation to the bottom

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,112 +114,107 @@ jsctl operator deploy --credentials /path/to/secret.json --version v0.0.1-alpha.
...
-in your cluster:
+Apply the installation to cluster when ready:
 
 ```shell
-jsctl operator installations apply --csi-driver
+kubectl apply -f installation.yaml
 ```
+##### Generate and apply Installation that configures Jetstack Secure components for Venafi TPP user
 
-##### Install the SPIFFE CSI Driver
```
Comment (irbekrm) [inline README.md:237]: https://github.com/jetstack/jsctl/pull/11#discussion_r970703640

File: docs/reference/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+## jsctl
```
Comment (charlieegan3) [inline docs/reference/README.md:1]: This file seems to be the same as docs/reference/jsctl.md - is that expected?

**Review by charlieegan3 — APPROVED** [review]
I think this looks good to me. 

It makes sense to reorder based on the current routes users are on through this tool.

Nit: do we have a duplicated docs/README.md ?

File: docs/reference/README.md (hunk 2 above)
Comment (irbekrm) [reply docs/reference/README.md:1]: Thanks for spotting that I removed that one

**Pull request merged**